themebuilder/cli/adapters/node-file-reader.js

```diff
@@ -0,0 +1,10 @@
+const fs = require("fs");
+
+module.exports = filename => {
+    return new Promise((resolve, reject) => {
+        fs.readFile(filename, "utf8", (error, data) => {
+            if(error) reject(error);
```

Comment:
I think conditions like above would be more readable if we move the true\false code block to a new line and enclose it by curly brackets

Reply:
Would not be in this case. Too many letters. But I'll rewrite.